src/panel-rooms.tsx

```diff
@@ -166,9 +166,14 @@ class RoomsPanel extends PSRoomPanel {
 	}
 	renderRoomList(title: string, rooms?: RoomInfo[]) {
 		if (!rooms || !rooms.length) return null;
+		const sortedRooms = rooms.sort((a, b) => {
+			if (!b.userCount || !a.userCount) return 0;
+			// ascending order
+			return b.userCount - a.userCount;
```

Comment:
```suggestion
			return (b.userCount || 0) - (a.userCount || 0);
```

Reply:
If we're going this direction, isn't it better if we one-liner it?

Reply:
Yes. But the suggested changes API is limited. Go ahead and remove the `return`

Reply:
Did.

Reply:
I would move the comment above the variable declaration instead of doing it inline.

Reply:
That is done.